geocoding/location-service/src/app/main.py

```diff
@@ -0,0 +1,68 @@
+import pymongo
+from flask import Flask, jsonify, request
+from os import environ
+
+from src.app.admins_fetcher import AdminsFetcher
+from src.app.fake_geocoder import FakeGeocoder
+from src.app.geocoder import Geocoder
+from src.app.geocoder_suggester import GeocodeSuggester
+from src.integration.mapbox_client import mapbox_geocode
+
+app = Flask(__name__)
+
+geocoders = []
+fake_geocoder = FakeGeocoder()
+
+if 'ENABLE_FAKE_GEOCODER' in environ:
+    geocoders.append(fake_geocoder)
+if 'MAPBOX_TOKEN' in environ:
+    access_token = environ['MAPBOX_TOKEN']
+    mongo_client = None
+    if 'DB_CONNECTION_STRING' in environ:
+        mongo_client = pymongo.MongoClient(environ['DB_CONNECTION_STRING'])
+
+    admins_fetcher = AdminsFetcher(access_token, mongo_client)
+    mapbox_geocoder = Geocoder(access_token, admins_fetcher)
+    geocoders.append(mapbox_geocoder)
+
+suggester = GeocodeSuggester(geocoders)
```

Comment:
I think it would be better to move this to a combination of [config](https://flask.palletsprojects.com/en/1.1.x/config/#configuration-best-practices) and [blueprints](https://flask.palletsprojects.com/en/1.1.x/blueprints/)

Reply:
Thanks! I've turned the fake geocoder into a blueprint that can be registered, that's a tidy way to do it. I haven't changed the config because AIUI you no longer set variables in the environment, you set variables in a file and then tell Flask where that file is through the environment. That would make the deployment for this service differ from the others, there may be benefit to that but at the moment I'm trying to keep this addition as minimally complex as possible.